api/internal/postgres/postgres.go

```diff
@@ -361,15 +361,22 @@ func FilteredManifestQuery(
 }
 
 /*
- * Execute a prepared query and scan the resulting rows into manifest objects
+ * A postgres-specific implemententation of the IndexClient
  */
-func ExecQuery(
-	connPool *pgxpool.Pool,
-	query    string,
-	args     ...interface{},
+type PgClient struct {
+	conn   *pgxpool.Pool
```

Comment:
Sure you want to name variable conn? "Conn" stands for "connection" to me, which is kinda misleading given this is a pool of connections